core/src/main/scala/chisel3/RawModule.scala

```diff
@@ -154,6 +180,44 @@ package object internal {
   /** Marker trait for modules that are not true modules */
   private[chisel3] trait PseudoModule extends BaseModule
 
+  /** Computes a name
+    *
+    * @param prefix The prefix associated with the seed
```

Comment:
is `prefix` used at all if `seed` is not set?   In other words, is is (defaultPrefix + defaultSeed) or (prefix + seed), but never a combination of the two?